src/java.desktop/share/classes/sun/awt/image/ImageRepresentation.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 1995, 2018, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2018, 2023 Oracle and/or its affiliates. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 1995, 2023, Oracle and/or its affiliates. All rights reserved.
```
You always update the second year leaving the first one intact. The comma after the year should remain.